src/Statistics/include/Statistics/BinomialConfidenceInterval.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <cstdint>
+
+namespace orbit_statistics {
+
+struct BinomialConfidenceInterval {
+  float lower;
+  float upper;
+};
+
+class BinomialConfidenceIntervalEstimator {
+ public:
+  [[nodiscard]] virtual BinomialConfidenceInterval Estimate(float ratio, uint32_t trials) const = 0;
```

Comment:
Nit: This could use a comment explaining that `ratio` is a probability and therefore needs to be between 0 and 1.